Apps/Src/Task_Init.c

```diff
@@ -0,0 +1,184 @@
+/* Copyright (c) 2020 UT Longhorn Racing Solar */
+#include "os.h"
+#include "config.h"
+#include "Tasks.h"
+#include "CAN_Queue.h"
+
+#ifndef SIMULATION
+#include "stm32f4xx.h"
+#endif
```

Comment:
Is simulation still supported?

Reply:
The simulator doesn't work right now. I think it would be nice if we could get it partially working again, so we could run some regression tests, but that's a problem for another time